### PR TITLE
UX: improves copy of various reports

### PR DESCRIPTION
**Pull request by jjaffeux**
- better differenciate webcrawlers reports
- s/private message/personal message

### CODE REVIEW

File: config/locales/server.en.yml [hunk 1]
```diff
@@ -1123,10 +1123,10 @@ en:
       yaxis: "Logged In Pageviews"
       description: "Number of new pageviews from logged in users."
     page_view_crawler_reqs:
-      title: "Web Crawlers"
+      title: "A count of all webcrawler pageviews"
```
Comment (davidtaylorhq) [inline config/locales/server.en.yml]:
```suggestion
      title: "Web Crawler Pageviews"
```

File: config/locales/server.en.yml [hunk 2]
```diff
@@ -1123,10 +1123,10 @@ en:
...
       xaxis: "Day"
       yaxis: "Web Crawler Pageviews"
-      description: "Webcrawlers listed by number of new pageviews they have generated."
+      description: "Number of pageviews generated by all webcrawlers each day."
```
Comment (davidtaylorhq) [inline config/locales/server.en.yml]:
```suggestion
      description: "Total pageviews from web crawlers over time."
```

File: config/locales/server.en.yml [hunk 3]
```diff
@@ -1182,11 +1182,11 @@ en:
       yaxis: "Number of visits"
       description: "Number of unique users who visited using a mobile device."
     web_crawlers:
-      title: "Web Crawler Requests"
+      title: "Webcrawler user agents with the most pageviews"
```
Comment (davidtaylorhq) [inline config/locales/server.en.yml]:
```suggestion
      title: "Web Crawler User Agents"
```

File: config/locales/server.en.yml [hunk 4]
```diff
@@ -1182,11 +1182,11 @@ en:
...
       labels:
         user_agent: "User Agent"
         page_views: "Pageviews"
-      description: "Number of new pageviews generated by all webcrawlers."
+      description: "List of webcrawler user agents ordered by pageviews."
```
Comment (davidtaylorhq) [inline config/locales/server.en.yml]:
```suggestion
      description: "List of web crawler user agents, sorted by pageviews."
```

**Comment by jjaffeux** [comment]
Yes I prefer it that way, thanks @davidtaylorhq , let's see what @coding-horror thinks  of it

**Comment by gschlager** [comment]
If we want to merge this, it should happen this week. We should refrain from updating translations next week to give translators some time to do their work.

**Pull request merged**